tests/Avalonia.RenderTests/Media/ImageBrushTests.cs

```diff
@@ -32,6 +32,23 @@ private string SmallBitmapPath
             get { return System.IO.Path.Combine(OutputPath, "github_icon_small.png"); }
         }
 
+        [Fact]
+        public async Task ImageBrush_NullSource()
+        {
+            Decorator target = new Decorator
+            {
+                Width = 200,
+                Height = 200,
+                Child = new Rectangle
+                {
+                    Margin = new Thickness(8),
+                    Fill = new ImageBrush()
+                }
+            };
+
+            await RenderToFile(target);
```

Comment:
This test isn't doing anything because you're not calling `CompareImages();` after rendering to the file.

You will also need to add the expected output images to the `tests\TestFiles` directory once you've done that in order for the tests to pass.

Reply:
Oh, sorry, I just oriented on exception in test. Sure, I'll add new file - it will be fully transparent png, 200*200.
Should I add same test to Scia project?

Reply:
Yes, you'll need to add it to both the D2D and Skia directories. If you add the comparison and run the test, it will tell you what files are missing.